### ISSUE TITLE
Improve serverlist refresh

### ISSUE THREAD

**Issue by i124q2n8** [teeworlds/teeworlds#1026]
There are some problems with the serverbrowser. 
1. Some servers could not be found if you have a slow connection due to packet loss. 
   ->retry timeouted servers
2. The reload leeds to a fps drop/lags. I think this caused by the sort function but i dont know.

(3.) Just an idea: reload the serverinfo you hovered every 10 seconds

What do you think about these 3 points?

**Comment by minus7** [comment]
Slightly related (for anyone who'd take on this task): #1281

**Comment by jtbx** [comment]
Is this still an issue? I don't experience lag on refresh or missing servers.

**Comment by ChillerDragon** [comment]
Yes this is still an issue. Reloading the whole list takes a while. And during that time only the freshly loaded servers are visible.
